feg/gateway/services/n7_n40_proxy/n7/n7.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package n7
+
+import (
+	"context"
+	"net/http"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+
+	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+)
+
+// createN7Client creates a N7 oapi client and sets the OAuth2 client credentiatls for authorizing requests
```

Comment:
You are missing `C`

Note the IDE should highlight the word differently if you did it right

![image](https://user-images.githubusercontent.com/16157139/142346342-0905644e-b114-4e81-bae0-0002dd826edb.png)